src/spring-cloud/azext_spring_cloud/_app_validator.py

```diff
@@ -76,3 +78,29 @@ def _get_active_deployment(client, resource_group, service, name):
         return next(iter(x for x in deployments if x.properties.active), None)
     except ResourceNotFoundError:
         raise InvalidArgumentValueError('Deployments not found under App {}'.format(name))
+
+
+def validate_deloy_path(namespace):
+    arguments = [namespace.artifact_path, namespace.source_path, namespace.container_image]
+    if all(not x for x in arguments):
```

Comment:
Is this check logic also for `validate_deloyment_create_path`? Otherwise, you assume `validate_deloyment_create_path` can use without any (--artifact-path, --source-path, --container-image).

Reply:
Yes, that is on purpose. The logic is having a default deployable for creation command like `app create` and `app deployment create`. But for `deploy` command, the executable file or the source code folder  must be provided.